Source/Modules/python.cxx

```diff
@@ -854,7 +863,14 @@ class PYTHON:public Language {
       Printv(f_shadow, "del version_info\n", NULL);
 
       if (builtin) {
-	Printf(f_shadow, "from %s import *\n", module);
+        /*
+         * Python3 removes relative imports.  So 'from _foo import *'
+         * will only work for non-package modules.
+         */
+        Printv(f_shadow, "if __name__.rpartition('.')[0] != '':\n", NULL);
+        Printf(f_shadow, tab4 "from .%s import *\n", module);
```

Comment:
Actually, my comment was for the one line only. In fact, I thought about something like this (to have context):
```c++
Printv(f_shadow, "if __name__.rpartition('.')[0] != '':\n", NULL);
Printv(f_shadow, tab4 "from %s%s import *\n", (py3 ? "." : ""), module );
Printv(f_shadow, "else:\n", NULL);
Printf(f_shadow, tab4 "from %s import *\n", module);
```

BTW, the module and package names are already known at "swigging" time. Then it's not necessary to generate the whole conditional python code to be executed at runtime. I think, the following should be enough (instead of the four lines above):

```c++
Printv(f_shadow, "from %s%s import *\n", (py3 && package && *Char(package)) ? "." : "", module );
```


You may also take a look at ``importDirective()`` and ``((abs|rel)_)?import_directive_string()`` functions.